### ISSUE TITLE
Help with Kohya Error 

### ISSUE THREAD

**Issue by MalloMalin** [kohya-ss/sd-scripts#809]
I managed to train a lora the day before yesterday and today I get this error 

train_network.py: error: unrecognized arguments: --cache_text_encoder_outputs
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ C:\Users\Mallo\AppData\Local\Programs\Python\Python310\lib\runpy.py:196 in _run_module_as_main   │
│                                                                                                  │
│   193 │   main_globals = sys.modules["__main__"].__dict__                                        │
│   194 │   if alter_argv:                                                                         │
│   195 │   │   sys.argv[0] = mod_spec.origin                                                      │
│ ❱ 196 │   return _run_code(code, main_globals, None,                                             │
│   197 │   │   │   │   │    "__main__", mod_spec)                                                 │
│   198                                                                                            │
│   199 def run_module(mod_name, init_globals=None,                                                │
│                                                                                                  │
│ C:\Users\Mallo\AppData\Local\Programs\Python\Python310\lib\runpy.py:86 in _run_code              │
│                                                                                                  │
│    83 │   │   │   │   │      __loader__ = loader,                                                │
│    84 │   │   │   │   │      __package__ = pkg_name,                                             │
│    85 │   │   │   │   │      __spec__ = mod_spec)                                                │
│ ❱  86 │   exec(code, run_globals)                                                                │
│    87 │   return run_globals                                                                     │
│    88                                                                                            │
│    89 def _run_module_code(code, init_globals=None,                                              │
│                                                                                                  │
│ in <module>:7                                                                                    │
│                                                                                                  │
│   4 from accelerate.commands.accelerate_cli import main                                          │
│   5 if __name__ == '__main__':                                                                   │
│   6 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│ ❱ 7 │   sys.exit(main())                                                                         │
│   8                                                                                              │
│                                                                                                  │
│ D:\Kohya and training\kohya_ss\venv\lib\site-packages\accelerate\commands\accelerate_cli.py:45   │
│ in main                                                                                          │
│                                                                                                  │
│   42 │   │   exit(1)                                                                             │
│   43 │                                                                                           │
│   44 │   # Run                                                                                   │
│ ❱ 45 │   args.func(args)                                                                         │
│   46                                                                                             │
│   47                                                                                             │
│   48 if __name__ == "__main__":                                                                  │
│                                                                                                  │
│ D:\Kohya and training\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py:918 in       │
│ launch_command                                                                                   │
│                                                                                                  │
│   915 │   elif defaults is not None and defaults.compute_environment == ComputeEnvironment.AMA   │
│   916 │   │   sagemaker_launcher(defaults, args)                                                 │
│   917 │   else:                                                                                  │
│ ❱ 918 │   │   simple_launcher(args)                                                              │
│   919                                                                                            │
│   920                                                                                            │
│   921 def main():                                                                                │
│                                                                                                  │
│ D:\Kohya and training\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py:580 in       │
│ simple_launcher                                                                                  │
│                                                                                                  │
│   577 │   process.wait()                                                                         │
│   578 │   if process.returncode != 0:                                                            │
│   579 │   │   if not args.quiet:                                                                 │
│ ❱ 580 │   │   │   raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)    │
│   581 │   │   else:                                                                              │
│   582 │   │   │   sys.exit(1)                                                                    │
│   583                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
CalledProcessError: Command '['D:\\Kohya and training\\kohya_ss\\venv\\Scripts\\python.exe', './train_network.py',
'--enable_bucket', '--min_bucket_reso=256', '--max_bucket_reso=2048', '--pretrained_model_name_or_path=D:/Kohya and
training/Training/1SDXL/sd_xl_base_1.0.safetensors', '--train_data_dir=D:/Kohya and training/Training/1SDXL/Axel\\img',
'--reg_data_dir=D:/Kohya and training/Training/1SDXL/Axel\\reg', '--resolution=1024,1024', '--output_dir=D:/Kohya and
training/Training/1SDXL/Axel\\model', '--logging_dir=D:/Kohya and training/Training/1SDXL/Axel\\log',
'--network_alpha=8', '--training_comment=Axel', '--save_model_as=safetensors', '--network_module=networks.lora',
'--text_encoder_lr=4e-05', '--unet_lr=0.0001', '--network_dim=8', '--gradient_accumulation_steps=3',
'--output_name=Axel_kompis', '--lr_scheduler_num_cycles=2', '--no_half_vae', '--learning_rate=0.0004',
'--lr_scheduler=cosine', '--lr_warmup_steps=100', '--train_batch_size=2', '--max_train_steps=996',
'--save_every_n_epochs=1', '--mixed_precision=fp16', '--save_precision=fp16', '--cache_latents',
'--cache_latents_to_disk', '--optimizer_type=AdamW8bit', '--optimizer_args', '--cache_text_encoder_outputs',
'--network_train_unet_only', '--bucket_reso_steps=32', '--max_data_loader_n_workers=0', '--bucket_reso_steps=64',
'--mem_eff_attn', '--gradient_checkpointing', '--xformers', '--bucket_no_upscale', '--noise_offset=0.0']' returned
non-zero exit status 2.

**Comment by dill-shower** [comment]
You should use sdxl_train_network.py

**Comment by MalloMalin** [comment]
> You should use sdxl_train_network.py

thank you for replying.
I'm not sure what that is. I think I did figure out what was wrong though.  When I loaded my configuration I believe the source model wasn't in there. When I picked source model all my other settings disappeared, when I noticed that I loaded again, looked them over and forgot that the model was wrong. 

I can't be sure anymore because I've gone back and fourth with loading and saving my configurations.
